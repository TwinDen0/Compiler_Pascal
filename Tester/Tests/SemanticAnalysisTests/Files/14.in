procedure x (y : integer);
begin

end;
begin
x('hi');
end.