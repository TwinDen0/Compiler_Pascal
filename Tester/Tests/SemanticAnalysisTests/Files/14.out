program
├─── procedure x
│    ├─── parameters
│    │    └─── integer
│    │         └─── y
│    ├─── local_types
│    ├─── begin
│    └─── end
├─── begin
│    └─── x
│         └─── "hi"
└─── end



Symbol Tables:
builtins:
	integer: SymInteger
	real: SymReal
	string: SymString
	write: SymProc
	read: SymProc
	exit: SymProc
globals:
	x: SymProc
		locals of procedure "x": 
		y: SymVar
